tests/conftest.py

```diff
@@ -20,10 +20,13 @@ def pytest_configure():
     """Perform initial configuration for pytest.
     Fetches pose data file paths as a dictionary for tests.
     """
-    pytest.POSE_DATA_PATHS = {
-        file_name: fetch_dataset_paths(file_name)["poses"]
-        for file_name in list_datasets()
-    }
+    pytest.DATA_PATHS = {}
+    for file_name in list_datasets():
+        paths_dict = fetch_dataset_paths(file_name)
+        if "poses" in paths_dict:
+            pytest.DATA_PATHS[file_name] = paths_dict["poses"]
+        elif "bboxes" in paths_dict:
+            pytest.DATA_PATHS[file_name] = paths_dict["bboxes"]
```

Comment:
interesting!

I found the suggestion a bit difficult to read tho (I am not that familiar with the walrus 🦭 ). But I liked the compactness so I had a go at a [variant](https://github.com/neuroinformatics-unit/movement/pull/231/commits/d28087c881d3f585492a9ad9dd8cf86dd933bb13). I think it reads a bit easier but let me know any thoughts.

Reply:
While the variant does read better, the `if` check is redundant, since a dict is always returned. Wdyt about:
```python
pytest.DATA_PATHS = {
    file_name: (paths_dict := fetch_dataset_paths(file_name)).get(
        "poses"
    )
    or paths_dict.get("bboxes")
    for file_name in list_datasets()
}
```
Alternatively, we stick to:
```python
pytest.DATA_PATHS = {}
for file_name in list_datasets():
    paths_dict = fetch_dataset_paths(file_name)
    data_path = paths_dict.get("poses") or paths_dict.get("bboxes")     
    pytest.DATA_PATHS[file_name] = data_path
```